GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -47,15 +47,14 @@ public QueryController(GitHubClient gitHubClient)
 
             return new AssignedChartResult
             {
-                milestone = milestone,
-                assignees = counts.Select(c => new AssigneeCount { assignee = c.Key, count = c.Value, }).OrderByDescending(a => a.count).ToArray(),
+                Milestone = milestone,
+                Assignees = counts.Select(c => new AssigneeCount { Assignee = c.Key, Count = c.Value, }).OrderByDescending(a => a.Count).ToArray(),
             };
         }
 
-        private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone)
+        private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone, string labels)
         {
-            if (string.IsNullOrWhiteSpace(milestone) ||
-                milestone == "any")
+            if (string.IsNullOrWhiteSpace(milestone) || milestone == "any" || milestone == "undefined" || milestone == "*")
```

Comment:
Where would you ever expect to get "undefined" from as a string?

Reply:
When the milestone is not set in the query params, the value will be 'undefined'.  This happens when only the milestone values are set.